### PR TITLE
TSK-1844: Refactor CreateTaskCommentAccTest to use test-api

### PR DESCRIPTION
**Pull request by ryzheboka**
https://sonarcloud.io/summary/new_code?id=ryzheboka_taskana&branch=TSK-1844

---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [x] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: lib/taskana-core-test/src/test/java/acceptance/taskcomment/create/CreateTaskCommentAccTest.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    int sizeBeforeInsertingComment = taskService.getTaskComments(taskWithComments.getId()).size();
+
+    taskService.createTaskComment(taskCommentToCreate);
+
+    List<TaskComment> taskCommentsAfterInsert =
+        taskService.getTaskComments(taskWithComments.getId());
+    assertThat(taskCommentsAfterInsert)
+        .contains(comment1, taskCommentToCreate)
+        .hasSize(sizeBeforeInsertingComment + 1);
+  }
```
Comment (ensarevlideveloper) [inline lib/taskana-core-test/src/test/java/acceptance/taskcomment/create/CreateTaskCommentAccTest.java:115]: I think, the assertion approach with containsExactlyInAnyOrder you used `should_CreateTaskComment_For_TaskThatAlreadyHasComments()` is better and smarter! You can reuse it.

File: lib/taskana-core-test/src/test/java/acceptance/taskcomment/create/CreateTaskCommentAccTest.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        taskService.newTaskComment("Definetely non existing ID");
+    newTaskCommentForNonExistingTask.setTextField("a newly created taskComment");
+    TaskComment newTaskCommentForTaskIdNull = taskService.newTaskComment(null);
+    newTaskCommentForTaskIdNull.setTextField("a newly created taskComment");
+
+    assertThatThrownBy(() -> taskService.createTaskComment(newTaskCommentForNonExistingTask))
+        .isInstanceOf(TaskNotFoundException.class);
+
+    assertThatThrownBy(() -> taskService.createTaskComment(newTaskCommentForTaskIdNull))
+        .isInstanceOf(TaskNotFoundException.class);
```
Comment (ensarevlideveloper) [inline lib/taskana-core-test/src/test/java/acceptance/taskcomment/create/CreateTaskCommentAccTest.java:130]: I think we want to check error messages too. So, we should do it here too. Consider building a `TaskNotFound Exception expectedException` so you just can use equals. See https://github.com/Taskana/taskana/pull/1946/commits/87c8b1ff7a5be74a1493cab0de3cbaabee456938 (after line 647)

**Review by ensarevlideveloper — COMMENTED** [review]
Nice to see that you've extended the class by more tests 👍

File: lib/taskana-core-test/src/test/java/acceptance/taskcomment/create/CreateTaskCommentAccTest.java (hunk 1 above)
Comment (ryzheboka) [reply lib/taskana-core-test/src/test/java/acceptance/taskcomment/create/CreateTaskCommentAccTest.java:115]: Ok :)

File: lib/taskana-core-test/src/test/java/acceptance/taskcomment/create/CreateTaskCommentAccTest.java (hunk 2 above)
Comment (ryzheboka) [reply lib/taskana-core-test/src/test/java/acceptance/taskcomment/create/CreateTaskCommentAccTest.java:130]: I will check the text of the error message :)

**Review by ensarevlideveloper — APPROVED** [review]
Thanks :)
